finout/manifest.json

```
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fc1d0074-ccd3-4513-a5c3-3d8ee99b1e49",
+  "app_id": "finout",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
```

```suggestion
    "Supported OS::Windows"
    "Offering::UI Extension"
```

Adding the offering type. You may also want to add categories here related to your offering. You can find the list of categories in the list at the top of this [docs page](https://docs.datadoghq.com/integrations/). To add a category, you'd use this syntax:

`"Category::Alerting"`